jenkins_jobs/trigger_weekly_host_os_build/post_build_script.sh

```diff
@@ -0,0 +1,14 @@
+GSA_USER_NAME="open-power-host-os"
+GSA_HOST_NAME="pokgsa.ibm.com"
+GSA_BUILDS_DIR="/gsa/pokgsa/projects/h/host-os/to_build"
+GSA_WEEKLY_DIR="/gsa/pokgsa/projects/h/host-os/weekly"
```

Comment:
This is hard-coded with the internal infrastructure.

Variables should not be prefixed with `GSA_`.